clients/src/main/java/org/apache/kafka/server/authorizer/AuthorizableRequestContext.java

```diff
@@ -17,11 +17,12 @@
 
 package org.apache.kafka.server.authorizer;
 
-import java.net.InetAddress;
 import org.apache.kafka.common.annotation.InterfaceStability;
 import org.apache.kafka.common.security.auth.KafkaPrincipal;
 import org.apache.kafka.common.security.auth.SecurityProtocol;
 
+import java.net.InetAddress;
```

Comment:
Unrelated change: can we revert this?

Reply:
Done. (Must sort out Kafka code style settings at some point)